python/ELASTICACHE_REDIS_CLUSTER_AUTO_BACKUP_CHECK/parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Version": "1.0",
+  "Parameters": {
+    "RuleName": "ELASTICACHE_REDIS_CLUSTER_AUTO_BACKUP_CHECK",
+    "SourceRuntime": "python3.6",
+    "CodeKey": "ELASTICACHE_REDIS_CLUSTER_AUTO_BACKUP_CHECK.zip",
+    "InputParameters": "{\"snapshotRetentionPeriod\": \"15\"}",
```

Comment:
Please move the key into OptionalParameter with an empty value
`"{\"snapshotRetentionPeriod\": \"\"}"`

Reply:
@jongogogo, its done!